defaults/main.yml

```diff
@@ -19,6 +23,12 @@ nginx_client_max_body_size: "64m"
 
 nginx_proxy_cache_path: ""
 
+nginx_extra_main_options: ""
```

Comment:
Could you rename this to `nginx_extra_conf_options` instead? That maps to the file name a little better, imo, and speaks to it being a little more generic.